lib/resolvers/event_mutations/unregisterForEvent.js

```diff
@@ -10,7 +10,9 @@ const unregisterForEventByUser = async (parent, args, context) => {
 
   if (!userFound) {
     throw new NotFoundError(
-      requestContext.translate('user.notFound'),
+      process.env.NODE_ENV !== 'production'
```

Comment:
import the variables in the constants file and replace the strings in this file with them, if you don't see it available, go and add it.